### PR TITLE
Merge `-lv` output into `stdout`

### PR DESCRIPTION
**Pull request by ZhangYet**
To fix https://github.com/bpftrace/bpftrace/issues/3379

##### Checklist

- [*] Language changes are updated in `man/adoc/bpftrace.adoc`
- [*] User-visible and non-trivial changes updated in `CHANGELOG.md`
- [*] The new behaviour is covered by tests

### CODE REVIEW

**Review by danobi — APPROVED** [review]
thanks! makes sense to me

File: src/probe_matcher.cpp [hunk 1]
```diff
@@ -480,7 +480,7 @@ void ProbeMatcher::list_probes(ast::Program* prog)
         std::cout << probe_type << ":" << match_print << std::endl;
         if (bt_verbose) {
           for (auto& param : param_lists[match])
-            LOG(V1) << "    " << param;
+            std::cout << "    " << param;
```
Comment (danobi) [inline src/probe_matcher.cpp:483]: Ah, missing a newline here. Nice tests.

**Review by danobi — COMMENTED** [review]
also while you're at it, do you mind updating commit message with the justification inline? it makes it easier to git-blame down the line without opening up github

File: src/probe_matcher.cpp (hunk 1 above)
Comment (ZhangYet) [reply src/probe_matcher.cpp:483]: added.

Comment (danobi) [reply src/probe_matcher.cpp:483]: Oh, I mean `std::endl` for the output

**Comment by ZhangYet** [comment]
> also while you're at it, do you mind updating commit message with the justification inline? it makes it easier to git-blame down the line without opening up github

Do you mean that I change the commit message to 
```
Merge `-lv` output into `stdout` to fix issue 3379
```

**Review by viktormalik — APPROVED** [review]
Good catch!

+1 on the better commit message but we can always adjust it when squashing the PR.

**Comment by ZhangYet** [comment]
@viktormalik @danobi Hi, I have fixed the commit message.

**Pull request merged**